Assignment2.js

```diff
@@ -0,0 +1,32 @@
+//2. What makes a method mutating or non mutating in Javascript? 
```

Comment:
Can you first explain here what is meant by mutating & non mutating

Reply:
Mutating methods are those that directly modify the original data structure.
Non-mutating methods do not modify the original data structure they are called on.